weave-js/src/components/PagePanelComponents/Home/Browse3/compare/ComparePage.tsx

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
any reason you put this (i suppose the entire compare dir) outside of the pages dir?

Reply:
At some point I'd like to do a more major reorganization of our UI directory structure. E.g. `PagePanelComponents` seems unnecessarily long when everything is already under `components`, it seems weird to have `Home` in the path to these, "Browse3" isn't a good name, etc.

"pages" seems redundant to me given all of this is under "PagePanelComponents" and I generally would rather have things organized/named by feature. But I can move it if you have a strong preference.

Reply:
I agree with the need to refactor. In my mind `weave-js/src/components/PagePanelComponents/Home/Browse3` is the root of our weave application and the initial refactor should be to just hoist that to a top level dir. Everything outside of that `Browse3` that is not common W&B should be either removed or extracted. Given that framing, the `pages` dir serves as the "topmost" dir for our pages.